packages/react/src/Form/Text/TextInput.tsx

```diff
@@ -26,7 +27,8 @@ export const TextInput = ({
   disabled = false,
   ...inputTextProps
 }: Props) => {
-  const inputId = useId();
+  const inputUseId = useId();
+  const inputId = id || inputUseId;
```

Comment:
Du coup, on a décidé de permettre la chaine vide
```suggestion
  const inputId = id ?? inputUseId;
```